src/main/resources/com/welovecoding/nbeditorconfig/Bundle.properties

```diff
@@ -1,7 +1,17 @@
 OpenIDE-Module-Name=EditorConfig Plugin
 OpenIDE-Module-Short-Description=EditorConfig Plugin for NetBeans IDE.
 OpenIDE-Module-Long-Description=\
-    A NetBeans IDE plugin supporting the EditorConfig standard. For more Information check: <a href="http://editorconfig.org/" target="_blank">http://editorconfig.org/</a> — After installation, simply restart your IDE to activate the plugin.\
+    A NetBeans IDE plugin supporting the EditorConfig standard. For more Information check: <a href="http://editorconfig.org/" target="_blank">http://editorconfig.org/</a> \u00e2\u0080\u0094 After installation, simply restart your IDE to activate the plugin.\
     <p>Plugin info: <a href="https://github.com/welovecoding/editorconfig-netbeans" target="_blank">https://github.com/welovecoding/editorconfig-netbeans</a></p>\
     <p><b>BETA DISCLAIMER</b><br/>THIS SOFTWARE IS IN BETA AND IS BELIEVED TO CONTAIN DEFECTS. A PRIMARY PURPOSE OF THIS BETA VERSION IS TO OBTAIN FEEDBACK ON SOFTWARE PERFORMANCE AND THE IDENTIFICATION OF DEFECTS. WE DO NOT RECOMMEND TO USE THIS SOFTWARE WITH PRODUCTION CODE. PLEASE MAKE BACKUPS OF IMPORTANT FILES BEFORE USING THIS SOFTWARE; USE CAUTION AND DO NOT TO RELY IN ANY WAY ON THE CORRECT FUNCTIONING OR PERFORMANCE OF THE SOFTWARE AND/OR ACCOMPANYING PARTS.</p>
 OpenIDE-Module-Display-Category=Editing
+
+text/x-editorconfig=EditorConfig
+
+comment=Comment
```

Comment:
Maybe we should prefix our property names, what do you think? Like they did with `OpenIDE-Module`.